components/date-picker/util.ts

```diff
@@ -160,3 +160,14 @@ export function getTimeProps<DateType, DisabledTime>(
     showTime: showTimeObj,
   };
 }
+
+export type MergeAllowClearProps = Pick<PickerProps<unknown>, 'clearIcon' | 'allowClear'>;
+
+export function mergeAllowClear(
+  { clearIcon, allowClear }: MergeAllowClearProps,
```

Comment:
It is recommended that you do not pass parameters as objects, as this is not conducive to code compression.

Reply:
Addressed (if my understanding was correct).